lib/device-log/ios-simulator-log.ts

```diff
@@ -32,16 +37,32 @@ export class IOSSimulatorLog extends LineConsumingLog {
     this.predicate = opts.iosSimulatorLogsPredicate;
     this.logLevel = opts.simulatorLogLevel;
     this.proc = null;
+    this.iosSyslogFile = opts.iosSyslogFile;
+    this.syslogLogger = null;
   }
 
   override async startCapture(): Promise<void> {
     if (_.isUndefined(this.sim.udid)) {
       throw new Error(`Log capture requires a sim udid`);
     }
-
     if (!(await this.sim.isRunning())) {
       throw new Error(`iOS Simulator with udid '${this.sim.udid}' is not running`);
     }
+    if (this.iosSyslogFile && this.showLogs) {
+      await this.clearExistingSyslog();
+      try {
+        this.syslogLogger = createLogger({
```

Comment:
Is there a way to provide the maximum file size for log rotation?  Syslogs are huge, so there is a point to enable their rotation by default to make sure they won't exceed the available free space on the host where the server is running.